flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/CastFunctionITCase.java

```diff
@@ -979,6 +1265,30 @@ private TestSpec build() {
                     .testResult(testSpecs.toArray(new ResultSpec[0]));
             return testSpec;
         }
+
+        private String tableApiSpecificErrorMsg(int i, LogicalType colType, String defaultMsg) {
```

Comment:
this looks like duplicate code to me that can be merged with the method below? also `i` is not used.